src/subscription_manager/gui/managergui.py

```diff
@@ -523,7 +523,7 @@ def _getting_started_item_clicked(self, widget):
         try:
             # unfortunately, Gtk.show_uri does not work in RHEL 5
             DEVNULL = open(os.devnull, 'w')
-            subprocess.call(["gnome-open", "ghelp:subscription-manager"],
+            subprocess.call(["xdg-open", "ghelp:subscription-manager"],
```

Comment:
Your proposed solution is better. I will test it on rhel6 and fedora and then I will squash commits.

BTW: it is possible to install xdg-utils package at RHEL6 too.